opc-ua-sdk/sdk-server/pom.xml

```diff
@@ -48,6 +48,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
```

Comment:
if and when this other PR is merged https://github.com/eclipse/milo/pull/1263 , the version can be removed